doc/user/content/concepts/indexes.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Indexes
+description: Conceptual page on indexes.
+menu:
+  main:
+    parent: concepts
+    weight: 20
+    identifier: 'concepts-indexes'
+---
+
+## Overview
+
+Indexes can [optimize query performance](/transform-data/optimization).  In
+addition, because indexes in Materialize are maintained in memory, indexing a
```

Comment:
See comment above about the "in-memory" claim.

Reply:
Sounds good! That's what we've had up in the documentation for months, in any case — no need to rush those updates.